lib/autoprefixer.js

```diff
@@ -88,7 +89,11 @@ module.exports = postcss.plugin('autoprefixer', (...reqs) => {
 
     plugin.options = options;
 
-    plugin.info = opts => require('./info')(loadPrefixes(opts));
+    plugin.info = function (opts) {
+        opts = opts || {};
+        opts.from = path.resolve('.');
```

Comment:
How about allowing defining `from` in the given `opts`, and only falling back to '.' if `opts.from` is empty? Would allow using this as a debug tool in more flexible manner if need be.